apps/site/components/Downloads/Release/PlatformDropdown.tsx

```diff
@@ -64,7 +64,7 @@ const PlatformDropdown: FC = () => {
       placeholder={t('layouts.download.dropdown.unknown')}
       ariaLabel={t('layouts.download.dropdown.installMethod')}
       onChange={platform => platform && release.setPlatform(platform)}
-      className="min-w-20"
+      className="min-w-28"
```

Comment:
```suggestion
      className="min-w-24"
```

Reply:
When changing this value to `28` to `24`, there seems to be a shift again;

https://github.com/user-attachments/assets/fff81fee-7c4a-4748-8ab0-ddf8222112f8

Reply:
I mean, shifts will always happen. See what happens when you force a Windows UA. Docker has an even bigger snippet 🤷 

Also, I wonder if we can add a default codebox and select value for when JavaScript is not available. Can we render a <noscript> version of a React component? 🤔

Reply:
I think we can do something with the [noscript](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript) element, but select may not be available 🤔 

If JavaScript is not available, how about adding a redirect button and a simple description to a page with download links?